.travis.yml

```diff
@@ -8,8 +8,8 @@ notifications:
   email: false
 before_script:
 - npm prune
-- lerna bootstrap --scope '@(function-tree|cerebral|cerebral-forms|cerebral-provider-http|cerebral-provider-firebase|cerebral-provider-forms|cerebral-router|babel-*)'
-- lerna bootstrap --scope @cerebral/codemods*
+- npm run setup:babel
```

Comment:
@Guria Using `npm run setup:babel` because you also included `|babel-*`.
Was not able to run 
```sh
lerna bootstrap --scope '@(function-tree|cerebral|cerebral-provider-http|cerebral-provider-firebase|cerebral-provider-forms|cerebral-router|cerebral-module-shortcuts|babel-*)'
```
Try it.